src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,161 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class RegistrationServiceImplTest {
+    private static List<User> userList;
+    private static User userWithInvalidAge;
+    private static User userWithValidData;
+    private static User userWithValidDateSameLogin;
+    private static User userWithInvalidPassword;
+    private static User userWithValidData1;
+    private static User userWithValidData2;
+    private static User userWithValidDateSameLogin2;
+    private static User userWithNullAge;
+    private static User userWithNullLogin;
+    private static User userWithNullPassword;
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @AfterEach
+    void tearDown() {
+        userList = Storage.people;
+    }
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
Does not need to be used before all user tests for all tests. What if you run one test?
Just create a user, and make the conditions you need in each test.